packages/paste-website/src/components/ContactUsMenu.tsx

```diff
@@ -5,8 +5,28 @@ import {Menu, MenuButton, MenuItem, useMenuState} from '@twilio-paste/menu';
 import {MoreIcon} from '@twilio-paste/icons/esm/MoreIcon';
 import {ProductSupportIcon} from '@twilio-paste/icons/esm/ProductSupportIcon';
 
-export const ContactUsMenu: React.FC<{}> = () => {
-  const menu = useMenuState({placement: 'right-end'});
+interface ContactUsMenuProps {
+  placement?:
```

Comment:
TODO: can we Pick<> the type from the Menu exports?

Reply:
From menu primitive yes, from menu no. We're not reexporting the types from the primitive in the menu. I'll create a ticket for it.